### ISSUE TITLE
[A11y][TFM] Tooltip icon present beside Frameworks label is defined as both 'frameworks filter information' link and tooltip info button.

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#9414]
Url: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1745398

Environment Details:
Application Name: Filtering by Target Frameworks
#URL: [NuGet Gallery | Packages (nugettest.org)](https://dev.nugettest.org/packages?q=)
Windows Version: Win11
OS Version: 22621.1105
Chromium Edge Version: Version 109.0.1518.78 (Official build) (64-bit)

Repro Steps: 
Hit the URL: [NuGet Gallery | Packages (nugettest.org)](https://dev.nugettest.org/packages?q=)
Tab to Frameworks section.
Tab to the tooltip icon present beside Frameworks label.
Press Enter and observe that tooltip info icon is behaving as a Link.
Again Tab to the same tooltip icon present beside Frameworks label .
Press Tab again.
Observe that now tooltip information appears on the same icon.

Actual Result:
Tooltip icon present beside Frameworks label is defined as both 'frameworks filter information' link and tooltip info button.

Expected Result:
Tooltip icon present beside Frameworks label should be defined only as Tooltip info button and 'frameworks filter information' link should be separated from that control.

Notes:
The observed keyboard interaction is a bit confusing. The cleanest thing to do there is likely going to be to separate the hover tooltip information and the link to additional information.
